src/vehicle_gui.cpp

```diff
@@ -3355,15 +3344,15 @@ void SetMouseCursorVehicle(const Vehicle *v, EngineImageType image_type)
 	bool is_ground_vehicle = v->IsGroundVehicle();
 
 	while (v != nullptr) {
-		if (total_width >= ScaleGUITrad(2 * (int)VEHICLEINFO_FULL_VEHICLE_WIDTH)) break;
+		if (total_width >= ScaleSpriteTrad(2 * (int)VEHICLEINFO_FULL_VEHICLE_WIDTH)) break;
 
 		PaletteID pal = (v->vehstatus & VS_CRASHED) ? PALETTE_CRASH : GetVehiclePalette(v);
 		VehicleSpriteSeq seq;
 
 		if (rotor_seq) {
 			GetCustomRotorSprite(Aircraft::From(v), image_type, &seq);
 			if (!seq.IsValid()) seq.Set(SPR_ROTOR_STOPPED);
-			y_offset = - ScaleGUITrad(5);
+			y_offset = - ScaleSpriteTrad(5);
```

Comment:
space between `-` and number (function call) is a bit odd (i know, it was already there)